docs/content/architectuur/zgw-in-gegevenslandschap.md

```diff
@@ -0,0 +1,188 @@
+Of horen sommige objecten echt bij elkaar? Wat is de relatie met het ImZTC? Om de eenduidigheid van de implementaties ten goede te komen, is het nodig om dit verder uit te werken.  Hoe vertalen we de objecten uit het RGBZ naar registraties in een gegevenslandschap en hoe relateren we deze aan elkaar? Met het informatiemodel laten we de implementatie nadrukkelijk vrij.
+In een gegevenslandschap is het wenselijk om dit verder uit te werken in een gegevensmodel. En wellicht ook met een concreet voorbeeld om dit concept aan iedereen goed duidelijk te maken.
+
+### 1.4.7. Wat willen we doen met Zaak?
+
+Zaakgericht werken kent al een lange geschiedenis. Het stamt uit de tijd dat dienstverleningsambities hoger werden en backoffice-applicaties dit niet of nauwelijks aankonden. Daarna was er de gedachte dat een “alles-in-1 zaaksysteem” voor veel processen voldoende zou zijn. Inmiddels draait de slinger een beetje terug en zien we taakspecifieke systemen moderner en opener worden. In een gegevenslandschap is dit helemaal het geval.
+
+In de loop der jaren zijn er diverse doelen voor zaakgericht werken ontstaan: dienstverlening verbeteren, als basis voor het archiveren, het bieden van managementinformatie,.. waarvan het de vraag is of je ze in een modern applicatielandschap nog met zaakgericht werken wilt oplossen. In het geval van een gemeente die met één dominant generiek zaaksysteem werkt ligt de oplossingsrichting voor de hand. In een gegevenslandshap veel minder. Bovendien zien we op vele vlakken een terugtrekkende beweging op het “gemeentebrede” dienstverlenigsconcept, meer richting domeinspecifieke Apps en portalen. Een meer domeingerichte dienstverlening en informatievoorziening (ipv generiek obv zaken) ligt dan meer voor de hand. Opmerking AK  Die twee zijn niet strijdig. Het lijkt er op dat in de redenering  en de voorbeelden het zaakgericht werken als panacee voor alle problemen en behoeften wordt gezien. Evenwel, zaakgericht werken is niet mee of minder dan een manier van procesgericht werken. Een manier die zo ver is uitgewerkt dat anderen dan de uitvoerenden van een proces dat proces ook kunnen begrijpen, op het niveau dat voor een ander volstaat: de hoofdlijnen. Dit ‘esperanto’ blijft een onderscheidend kenmerk van zaakgericht werken en zaken. Je krijgt daarmee twee niveau’s: domeinspecifiek en domeinoverstijgend. Het tweede doen we met zaken.
+
+Enkele voorbeelden:
+1. Een gemeente wil alle Meldingen Openbare Ruimte op een kaart tonen. Vraag je hiertoe alle melding-zaken op met een generiek ZDS-koppelvlak? Wetende dat de AVG dit voor het overgrote deel aan zaken/zaaktypen niet toestaat? Of kies je voor een meldingen-specifiek koppelvlak, dat mogelijk functioneel rijker is: type melding, prioriteit van de melding, tekstuele toelichting…allemaal wellicht nuttig om op een dergelijke kaart te kunnen weergeven. Maar niet of slechts met moeite (zaaktypespecifieke eigenschappen) met een generiek zaken-koppelvlak te realiseren.
```

Comment:
De combinatie van ZAAK en MELDING object. Dat betekent dat we MELDING ook moeten uitmodelleren.

Reply:
mee eens. en INZAGEVERZOEK en SUBSIDIEAANVRAAG en ...
wellicht zitten daar generieke elementen in